app/client/models/TreeModel.ts

```diff
@@ -110,6 +111,7 @@ export function fromTableData(table: TreeTableData, buildDom: DomBuilder, oldMod
     delete indentations[rec.indentation + 1];
     const item = oldItems[rec.id] || new TreeItemRecord();
     item.hidden = rec.hidden;
+    item.collapsed = rec.collapsed;
     item.init(storage, index, reverse(children));
     item.buildDom = () => buildDom(rec.id, rec.hidden);
```

Comment:
To make information about children available to `buildDom`:
```suggestion
    item.buildDom = () => buildDom(rec.id, item);
```